topic04-git-introduction/book-git-tutorial/02.Branching.md

```diff
@@ -0,0 +1,34 @@
+# Branching With GitHub
+
+----
+## What is Branching?
+
+
+This document is to show how branching works in git and how to use it. Branching is a feature available in git that lets you pull off from the master branch to make changes that won't directly effect the master branch unless merged. It also means that others have to review your work first just in case you have errors in your work such as spelling or bugs. 
```

Comment:
Good. I would suggest having a look at [this article on branching](https://www.atlassian.com/git/tutorials/using-branches) to see what branching covers, and the areas you may want to cover in your tutorial